gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/test/java/io/gravitee/rest/api/management/v2/rest/resource/documentation/ApiPagesResourceTest.java

```diff
@@ -71,6 +89,9 @@ class ApiPagesResourceTest extends AbstractResourceTest {
     @Autowired
     private UserCrudServiceInMemory userCrudService;
 
+    @Autowired
+    DocumentationValidationDomainService documentationValidationDomainService;
```

Comment:
Why add the domain service here? Should be only the InMemory services added at this level.

Reply:
It's an error I removed it. my bad